zeppelin-zengine/pom.xml

```diff
@@ -42,6 +42,18 @@
     </dependency>
 
     <dependency>
+      <groupId>${project.groupId}</groupId>
```

Comment:
Indeed, Zengine should not have such dependency. 

@aspen01 if `HDFSCommand` have to be shared:
 - one option is to move it to `zeppelin-interpreter`, as both `file` and `zeppelin-zengine` already depend on it
 - another option to consider would be - extract a new maven sub-module out of ` zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/`, move `HDFSCommand` there and make other modules depend on it.

Second option is more involved and deserves a separate issue \w discussion. First one looks more feasible